docs/IRISMapCube.rst

```diff
@@ -0,0 +1,264 @@
+
+    >>> my_cube.scaled
+    True
+
+Dimensions
+----------
+
+As ``IRISMapCube`` is inherited from NDCube_, we can use the two properties of NDCube_
+which allow us to get the data shape and the axis types of our IRISMapCube object. These
+properties are described in the NDCube.Dimensions_ section.
+
+Cropping and Indexing
+---------------------
+
+One of the most powerful capabilities of IRISMapCube, coming from NDCube_, is the slicing
+ability. There are two ways to slice: using array-like indices or by coordinates. These are described in the NDCube.Slicing_ section.
+
+Manipulating the Data
+---------------------
+
+We can manipulate an IRISMapCube object with the methods listed below.
+
+.. _Exposure_Time_Correction:
+
+Exposure Time Correction
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This method scales the data from data number (DN) units to DN per second, thereby correcting for any changes in exposure time during an observation and allowing a better comparison between different observations. To use it, we do: ::
+
+    >>> my_cube.apply_exposure_time_correction()
+
+We also can undo the exposure time correction by doing: ::
+
+    >>> my_cube.apply_exposure_time_correction(undo=True)
+
+The correction is only applied (undone) if the object's unit doesn't (does) already
+include inverse time. This can be overridden so that correction is applied (undone)
+regardless of unit by setting ``force=True``. Use one of the two lines above to apply
+(undone) by using the force kwarg: ::
+
+    >>> my_cube.apply_exposure_time_correction(force=True)
+    >>> my_cube.apply_exposure_time_correction(undo=True, force=True)
+
+.. _Dust_Particle_Mask:
+
+Dust particle mask
+^^^^^^^^^^^^^^^^^^
+
+Some IRIS slit-jaw image pixels are obscured by dust, and no data is available at those locations. The ``apply_dust_mask`` method of ``IRISMapCube`` can be used to mask these dust pixel locations by adding them to the invalid pixel mask. We can use this method by doing: ::
+
+    >>> my_cube.apply_dust_mask()
+
+Now, our ``my_cube.mask`` contains the dust particles positions and we can use it to
+select only the data we want. If we want to remove the dust particle positions from
+our mask, we can call again this method with the ``undo`` kwarg. ::
+
+    >>> my_cube.apply_dust_mask(undo=True)
+
+If unsure if the the dust particle mask is applied
+to our ``my_cube.mask``, we can check the ``dust_masked`` attribute of our object: ::
+
+    >>> my_cube.dust_masked
+
+If ``True``, the dust particle positions are added to our ``my_cube.mask``, if
+``False`` the dust particle positions are not added.
+
+Visualisation
```

Comment:
American spelling should be used in accordance with the SunPy style rules, i.e. ```Visualization```.